libs/search-widget/src/tiles/pdf-tile/PdfTile.svelte

```diff
@@ -171,6 +171,19 @@
 <svelte:window bind:innerWidth
                on:keydown={handleKeydown}
                on:resize={(event) => resizeEvent.next(event)}/>
+<svelte:head>
+  <script src="https://cdn.jsdelivr.net/npm/pdfjs-dist@{pdfJsVersion}/build/pdf.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/pdfjs-dist@{pdfJsVersion}/web/pdf_viewer.js"></script>
+  <link  href="https://cdn.jsdelivr.net/npm/pdfjs-dist@{pdfJsVersion}/web/pdf_viewer.css" rel="stylesheet">
```

Comment:
It seems the styles are loaded but are not being applied to the pdf viewer.
It only happens when using the widget as a web component, maybe it's related to the shadow DOM.

Reply:
ok, thanks. I'll check again to fix this